charts/cert-manager-release/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: cert-manager-release
+description: A chart that bundles cert-manager CRDs with cert-manager helm chart
+type: application
+version: v1.7.1
+appVersion: v1.7.1
+
+dependencies:
+  - name: cert-manager
+    version: v1.7.1
+    repository: https://charts.jetstack.io
```

Comment:
What I generally don't like about Helm chart dependencies is that they are basically invisible from outside (e.g. `helm ls` doesn't show them). Can we not make this a mere Kustomization that installs a cert-manager HelmRelease and then have another HelmRelease depend on it? That's what we do with [kubefed](https://github.com/mesosphere/kommander-applications/tree/20ca18fb41f6742c9013f88fe5bef42c1da4030d/services/kubefed/0.9.1), for example.

Reply:
How does the linked kubefed install 2 depending helmreleases?

Reply:
This will be handled in new @mikolajb 's PR and he will try to not embed charts.